### PR TITLE
user model created

### PR DESCRIPTION
**Pull request by danielnottingham**
user model created
@edimossilva

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -10,11 +10,11 @@
...
-  adapter: mysql2
-  encoding: utf8mb4
+  adapter: postgresql
+  encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  username: root
-  password:
+  username: postgres
+  password: 123
   host: localhost
```
Comment (edimossilva) [inline config/database.yml:19]: Create a issue tô use environment variables to user and password

File: test/fixtures/users.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
```
Comment (edimossilva) [inline test/fixtures/users.yml:2]: Remove this file
Create a issue tô not generate fixtures

File: test/models/user_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
```
Comment (edimossilva) [inline test/models/user_test.rb:5]: Remove this file
Create a issue tô not use minitest as default

File: test/fixtures/users.yml (hunk 2 above)
Comment (danielnottingham) [reply test/fixtures/users.yml]: file removed

File: test/models/user_test.rb (hunk 3 above)
Comment (danielnottingham) [reply test/models/user_test.rb]: file removed

File: config/database.yml (hunk 1 above)
Comment (danielnottingham) [reply config/database.yml:19]: I tried to edit the environment variables as I searched the internet, but it didn't work.
Ex: EDITOR=vim rails credentials:edit 
Couldn't decrypt config/credentials.yml.enc. Perhaps you passed the wrong key?


I tried without using the dotenv or figaro gem

Comment (edimossilva) [reply config/database.yml:19]: "Create a issue" you should not do It now, you should only create a issue with this topic

**Comment by edimossilva** [comment]
delete this branch

**Pull request merged**